tests/Fixtures/3LineItems.yml

```diff
@@ -0,0 +1,12 @@
+OAT\SimpleRoster\Entity\LineItem:
+    line_item_{1..2}]:
+            label: 'line_item_<current()>'
+            uri: 'https://test.taocloud.fr/__n/<current()>/'
+            slug: 'slug-<current()>'
+    line_item_qqy:
```

Comment:
Does `qqy` have any meaning or just random?

Reply:
yes, It is :)